ui/pages/permissions-connect/permissions-connect.container.js

```diff
@@ -50,9 +50,10 @@ const mapStateToProps = (state, ownProps) => {
     } else {
       const targetUrl = new URL(origin);
       targetSubjectMetadata = {
-        host: targetUrl.host,
```

Comment:
This property is no longer part of subject metadata, and isn't used in this part of the component tree anyway.